### PR TITLE
create custom discord notification for bugs

### PR DESCRIPTION
**Pull request by EdersonRodrigues7**
closes https://github.com/codante-io/roadmap/issues/72

### CODE REVIEW

**Review by robertotcestari — APPROVED** [review]

File: config/discord.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 return [
     'channels' => [
-        'notificacoes' => 'https://discord.com/api/webhooks/1106953489839566959/te4Btbt_J3iE-J-TpJp7zBc4jo_1cBhvekCwZ4JN-yhIsXPrZBtsTnoi4VTI7bk8mvOp'
+        'notificacoes' => 'https://discord.com/api/webhooks/1106953489839566959/te4Btbt_J3iE-J-TpJp7zBc4jo_1cBhvekCwZ4JN-yhIsXPrZBtsTnoi4VTI7bk8mvOp',
```
Comment (robertotcestari) [inline config/discord.php:5]: image

Comment (robertotcestari) [inline config/discord.php:5]:
```suggestion
        'notificacoes' => env('DISCORD_WEBHOOK_NOTIFICACOES'),
```

File: config/discord.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
+        'bugs' => 'https://discord.com/api/webhooks/1113459877419962438/zJM_NnS5tZc0eFR71TOWaSSuGf85xSwvQ1Qtre9VD082BizdLv2s6j3D8caCbnfxnq3r'
```
Comment (robertotcestari) [inline config/discord.php:6]:
```suggestion
        'bugs' => env('DISCORD_WEBHOOK_BUGS')
```

**Review by robertotcestari — CHANGES_REQUESTED** [review]

**Review by EdersonRodrigues7 — COMMENTED** [review]
@robertotcestari pode mergear pfvr

**Comment by robertotcestari** [comment]
Done

Em qua., 31 de mai. de 2023 às 15:16, Ederson Rodrigues <
***@***.***> escreveu:

**Pull request merged**